frontend/src/metabase/querying/filters/components/DatePicker/RelativeDatePicker/DateOffsetIntervalPicker/DateOffsetIntervalPicker.unit.spec.tsx

```diff
@@ -300,12 +300,8 @@ describe("DateOffsetIntervalPicker", () => {
         expect(screen.getByText(/months (ago|from now)/)).toBeInTheDocument();
         expect(screen.getByText(/quarters (ago|from now)/)).toBeInTheDocument();
         expect(screen.getByText(/years (ago|from now)/)).toBeInTheDocument();
-        expect(
-          screen.queryByText(/hours (ago|from now)/),
-        ).not.toBeInTheDocument();
-        expect(
-          screen.queryByText(/days (ago|from now)/),
-        ).not.toBeInTheDocument();
+        expect(screen.getByText(/hours (ago|from now)/)).toBeInTheDocument();
```

Comment:
Should we sort the assertions in order of increasing size?

Reply:
ce51e39423690aea6c0c6a5f4eab0f85e0668407